apps/google-analytics-4/frontend/src/helpers/contentTypeHelpers/contentTypeHelpers.ts

```diff
@@ -37,17 +41,19 @@ export const generateEditorInterfaceAssignments = (
   return { ...newAssignments, ...assignmentsToAdd };
 };
 
+// only include short text and short text list fields in the slug field dropdown
+const isCompatibleFieldType = (field: ContentTypeField): boolean => {
+  const isArray = field.type === 'Array';
+  return field.type === 'Symbol' || (isArray && (field.items as FieldItem).type === 'Symbol');
```

Comment:
Does this need to be `FieldItem[]` referencing the list?

Reply:
What should the behavior be if one of the items isn't the type symbol? Should we first filter it out?

Reply:
If you look at the `Array Fields` section of [this page](https://www.contentful.com/developers/docs/concepts/data-model/) in the docs, you'll see that the items property contains one type per field. We handle the inclusion of short text list fields this way in the ecommerce base app: https://github.com/contentful/apps/blob/master/packages/ecommerce-app-base/src/AppConfig/fields.ts#L39-L42

Open to other suggestions @matthew-gordon and @ryunsong-contentful if there's a better way to approach this!